### ISSUE TITLE
Tasks without priority are not visible/searchable

### ISSUE THREAD

**Issue by Maybach91** [mayankmkh/linear-app-plugin#65]
If i’m looking for a specific Task Number i cannot find it. Its not there. There are only like 50-100 Tasks, but i have a Linear Team with about 1000 Tasks (which are not all open tho). 

`96` should list all Tasks from 960 to 969, right?
![image](https://user-images.githubusercontent.com/1867543/140294487-9f3a9a83-a813-43f0-842a-ccaee913d871.png)

I deactivated the Cache and set the history up to 200, and restarted PhpStorm, but still no tasks visible.

**Edit:** 
The task got visible when i set a Priority. Tasks without priority are not visibble.

**Comment by mayankmkh** [comment]
Thanks for reporting the issue @Maybach91 and sorry for not replying earlier. 
I looked into the issue and found the search was not implemented because Linear didn't provide the API to do so. In the current version, the IDE is searching locally from the list of tasks it fetched which are ordered by the task's updated time. I think that's why the task got visible after priority was updated. But now there is an API available (in the alpha stage) which I'll try to add.
